changelogs/unreleased/5211-rajatvig-small.md

```diff
@@ -0,0 +1 @@
+Contour now sets GOMAXPROCS to match the number of CPUs available to the container which results in lower and more stable CPU usage under high loads and where the container and node CPU counts differ significantly.
```

Comment:
small change in wording but something to the effect of saying this is the new default behavior and you can still override it by setting `GOMAXPROCS` yourself would be good

Reply:
Does the update look good?